spharpy/samplings/coordinates.py

```diff
@@ -310,3 +310,17 @@ def n_max(self, value):
             self._n_max = None
         else:
             self._n_max = int(value)
+
+    @property
+    def quadrature(self):
```

Comment:
from what I understood last time the quadrature should be set during intitialization e.g. in the samplings module. If we derive it from the weights it does not contain any extra information. If I add weights later by voronoi, it will result in wrong result here.

Reply:
the SamplingSphere class is just a conatiner for SamplingSpheres, so for every grid with correct weight it should sum up to 4pi, independed of beeing a quadrature or not. The SamplingSphere is for example generated in samplings module, where we know each time if it is a quadrature or not. it does not make sence in my optionon to guess later on if it is quadrature or not, especially if we know it for sure at a earlier stage. This can cause much more errors, becuase users will assume that the property is giving the right result. 
Also Users should just use the class if they have a samplingSphere and not any coordinates, so they should know if their sampling is a quadrature or not. we could use the default as False, then nothing will go wrong in using it in the SH class with defaults

Reply:
My understanding was that these are exactly the requirements for numerical integration (as used in the inverse transform), no matter if the weights are analytically computed or using Voronoi tesseleation.

Reply:
I don't think that implementing it like this is the best idea. Quadrature should only be true if the grid is designed to match a feasible quadrature rule. In other cases the weights can be used for weighted least-squares.
See https://pub.dega-akustik.de/NAG_DAGA_2009/data/articles/000296.pdf